src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -163,6 +163,48 @@ public ResponseEntity<Measure> updateMeasure(
     return response;
   }
 
+  @PutMapping("/measures/{id}/delete")
+  public ResponseEntity<Measure> deactivateMeasure(
+      @PathVariable("id") String id,
+      @RequestBody Measure measure,
+      Principal principal,
+      @RequestHeader("Authorization") String accessToken) {
+    ResponseEntity<Measure> response;
+    final String username = principal.getName();
+    if (id == null || id.isEmpty() || !id.equals(measure.getId())) {
+      log.info("got invalid id [{}] vs measureId: [{}]", id, measure.getId());
+      throw new InvalidIdException("Measure", "Update (PUT)", "(PUT [base]/[resource]/[id])");
+    }
```

Comment:
this check doesn't makes much sense. instead I would use  
```suggestion
    if (StringUtils.isBlank(id)) {
      log.info("got invalid id [{}]", id);
      throw new InvalidIdException("Invalid measure id: " + id);
    }
```